autosklearn/data/feature_validator.py

```diff
@@ -327,7 +325,7 @@ def get_feat_type_from_columns(
                 else:
                     raise ValueError(
                         "Input Column {} has unsupported dtype {}. "
-                        "Supported column types are categorical/bool/numerical dtypes. "
+                        "Supported column types are categorical/bool/numerical/string dtypes. "  # noqa: E501
```

Comment:
I don't think this is necessary. Please reformat the string so the lines fit within the line limit.

Reply:
i change it